templates/dex.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: kubeaddons.mesosphere.io/v1alpha1
+kind: Addon
+metadata:
+  name: dex
+  namespace: kubeaddons
+spec:
+  kubernetes:
+    minSupportedVersion: v1.14.0
+  chartReference:
+    chart: stable/dex
+    version: 1.4.0
+    values: |
+      ---
+      image: mhrabovcin/dex
+      imageTag: "d2f240c9c72682913eaff4fd85d119d7d2d0f240"
```

Comment:
Might be a nice cordiality to add a comment above this with the entire URL pointing to this revision, just makes it easy for someone to click while viewing the code. What do you think?

Reply:
can we get a version tag even if it is temporal ?

Reply:
Indeed, it would be nice to just tag anything we're going to use for this purpose for the time being.